lib/puppet/provider/service/debian.rb

```diff
@@ -16,7 +16,8 @@
   # is resolved.
   commands :invoke_rc => "/usr/sbin/invoke-rc.d"
 
```

Comment:
If a user wants to continue managing a service with this debian flavor of init, they would be unable to. Especially because of the bug in systemd on jessie when managing services with sysv profile (see PUP-4503 for more details), it would be disadvantageous to users to completely prevent them from using this provider.